scripts/workspace-conversion/README.md

```diff
@@ -0,0 +1,60 @@
+## Converting Workspace Dataset Attributes to Snapshot Data
+
+# Description
+This script takes the workspace dataset attributes and attempts to map them to similar fields in the data catalog service as snapshots.
+
+# Dataset Attributes
+An example workspace with attributes can be found [here](src/data/workspace-cleanAttributes.js.displayLibraryAttributes).
```

Comment:
Do you mean to link to https://github.com/DataBiosphere/terra-ui/blob/dev/src/data/workspace-attributes.js ?

Reply:
Oh thank you, this is definitely a better choice